documents/Examples/BxDF/standard_surface.mtlx

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<materialx version="1.36">
+  <!--
+    Autodesk Standard Surface
+  -->
+  <nodedef name="ND_standard_surface_surfaceshader" node="standard_surface" type="surfaceshader" nodegroup="pbr"
```

Comment:
Moved from being a pbrlib member to an external example built from pbrlib nodes.

Reply:
Excellent!